tests/slack_sdk/web/test_web_client_issue_971.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+from slack_sdk.web import WebClient
+from tests.slack_sdk.web.mock_web_api_server import (
+    setup_mock_web_api_server,
+    cleanup_mock_web_api_server,
+)
+
+
+class TestWebClient_Issue_971(unittest.TestCase):
```

Comment:
@eddyg Thanks for promptly adding this test! For verifying if the fix in this PR eliminates the issue in #971, we should have a str argument pattern(= this means do `json.dumps` as the issue report does) for attachments (and also blocks for the future - just in case).

Reply:
Great idea. Let me know what you think of the updated test suite.